core/templates/domain/story/StoryObjectFactory.ts

```diff
@@ -245,16 +237,34 @@ export class StoryObjectFactory {
     );
   }
 
-  /**
- * TODO(#14169): Remove the interstitial story so that full story can be
- * created from start.
- */
   // Create an interstitial story that would be displayed in the editor until
   // the actual story is fetched from the backend.
   createInterstitialStory(): Story {
+    let sampleStoryContentsBackendDict = {
+      initial_node_id: '',
+      nodes: [
+        {
+          id: '',
```

Comment:
An empty string is not a good default value.

Reply:
I have updated the solution as per the suggestion received. PTAL, thanks!